backends/platform/3ds/osystem-graphics.cpp

```diff
@@ -36,6 +36,8 @@
 
 namespace _3DS {
 
+extern MagnifyMode magnifyMode;
```

Comment:
It's better to store this in the `OSystem_3DS` rather than having an extern declaration.